dashboard/app/controllers/home_controller.rb

```diff
@@ -113,6 +113,9 @@ def init_homepage
     @homepage_data[:mapboxAccessToken] = CDO.mapbox_access_token
     @homepage_data[:currentUserId] = current_user.id
 
+    current_user_permissions = UserPermission.where(user_id: current_user.id).pluck(:permission)
+    @homepage_data[:showStudentAsVerifiedTeacherWarning] = current_user.student? && current_user_permissions.include?(UserPermission::AUTHORIZED_TEACHER)
```

Comment:
nit: you can use [`current_user.authorized_teacher?`](https://github.com/code-dot-org/code-dot-org/blob/92493cc72533cb1b99ed59213a542c80b283e220/dashboard/app/models/user.rb#L1349) here

Reply:
I think you mean `current_user.verified_teacher?`, that's what I did originally and then I found out that it calls https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/concerns/user_permission_grantee.rb#L17-L26 which will immediately return false if the account is a student account, so that doesn't work for this case

Reply:
Ah so it does. Does that mean that authorized teacher features don't work for student accounts already (this question doesn't block this PR or anything, just curious)

Reply:
The original bug report came from EP Tools team "we are currently updating scripts with instructor and participant types which could get us in a funny state where a student can be enrolled in a PL course", so I'm assuming this was manifesting in some way for them? But that's a good question.. I'm not 100% sure

Reply:
I'll bring this up with EP tools though, it's probably something they should be aware of